duradmin/src/main/webapp/js/spaces-manager.js

```diff
@@ -8,7 +8,7 @@
 
```

Comment:
It looks like there are only one or two functional changes here (as in the spaces-manager.jsp). It would be better I think to include the functional changes in this PR.  Then group the formatting changes in a separate PR.